client/components/PopUp.jsx

```diff
@@ -69,65 +66,94 @@ const CssTextField = withStyles({
   },
 })(TextField);
 
-
-
-
 const PopUp = (props) => {
   const classes = useStyles();
   const [text, addText] = useState('');
+  const [eventDate, addEvent] = useState(new Date());
   const [modalStyle] = React.useState(getModalStyle);
   const [open, setOpen] = useState(true);
+  const [formInfo, setFormInfo] = useState({
+    username: 'purpleswan857',
+    time: props.date.substring(0,16)
+  });
 
   let dateString = '';
 
-  if(props.date !== undefined) {
-    dateString = props.date.substring(0,props.date.length);
-  }
-
-
-  const handleOpen = () => {
-    setOpen(true);
 
-  };
+  if (props.date !== undefined) {
+    dateString = props.date.substring(0,16);
+    //visualDate = props.visualdate ....
+  }
 
   const handleClose = () => {
     setOpen(false);
     props.callback(false);
   };
 
   const handleChange = (event) => {
-    addText(event.target.value);
 
+    const obj = {
+      [event.target.name]: event.target.value,
+    }
+    setFormInfo(formInfo => {
+      formInfo[event.target.name] = event.target.value
+      return formInfo;
+  });
+    //addText(event.target.value);
   }
 
   const handleSubmit = (event) => {
+    event.preventDefault();
     console.log('submitted');
-    axios.post('/post', {
-      date: props.date,
-      text: text
-      //what do we need to send a post request?
-      //date, industry, message, remote,
+    axios.post('/notification/addnotification', formInfo)
+    .then((response) => {
+      console.log(response);
     })
+    .catch((err) => {
+      console.log(err);
+    })
+  }
+
+  const handleDate = (event) => {
+    addEvent();
   }
 
   const body = (
     <div style={modalStyle} className={classes.paper}>
       <p>{dateString}</p>
-      <form onChange = {handleChange}>
-        <CssTextField value = {text} id="outlined-basic" label="please add event" variant="outlined" />
+
+      <form onSubmit={handleSubmit}>
+        <CssTextField className = {classes.bottom} onChange={handleChange} id="standard-basic" label="Add title" name = 'title'/>
```

Comment:
minor styling comment, but usually in React, attributes are formatted like `<span className={className}>Menu</span>` with no spaces around the =